### ISSUE TITLE
Amount of spaces causes inline operator overloading to fail

### ISSUE THREAD

**Issue by kungfooman** [enkimute/ganja.js#123]
I just run into an issue with code like this:

```js
      (l3 & l4) * (l1 & l2) * b * (l1 & l2) * (l3 & l4), 'b\'\'',
      (l3 & l4) * (l1 & l2) * b  * (l1 & l2) * (l3 & l4), 'b\'\'',
```

It looks nearly identical, but the amounts of spaces (1 space vs 2 spaces) generates different behaviour:

![image](https://user-images.githubusercontent.com/5236548/114310530-45902a00-9aeb-11eb-8bdf-8330080429fc.png)

**Comment by broundtree** [comment]
I think I have also run accross this problem... very confusing as to why code that works correctly crashes the app. My code sample is here on line 65. Just add one sauce where indicated and run, it fails.
https://enkimute.github.io/ganja.js/examples/coffeeshop.html#7luxYQzPN

BTW: GREAT APP!!!!!

**Comment by kungfooman** [comment]
Right, the integrated Ganjascript parser isn't the most reliable piece of software :sweat_smile: I run into more issues and IMO the best way forward would be to resort to using a maintained and extendable JS parser library (I have most experience with `@babel/parser`, so I would probably go with that).

However Babel parser comes with around 0.5 MB extra download (unminified)...

Acorn is smaller at least:

https://www.npmjs.com/package/acorn/v/7.4.1?activeTab=code

![image](https://github.com/enkimute/ganja.js/assets/5236548/e41fb3da-fa2e-4487-b0c1-925a88d1e0fc)

And that's not minified yet aswell - adding minification and gzip - it's like the size of a small JPG.
